packages/tools/devtools/devtools-view/src/test/ContainerSummaryView.test.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+} from "@fluidframework/devtools-core/internal";
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
```

Comment:
We may want to just update the package's eslint config to allow importing `@testing-library/jest-dom` this way.

https://github.com/import-js/eslint-plugin-import/blob/6d34c88a91a9cb7556700b7cb83c8a27731ff302/docs/rules/no-unassigned-import.md#options

Reply:
I thought the same thing but the option is a glob for _files_, not for _packages_ 😢. We could ignore all test files but I wasn't in love with that approach.

Reply:
... or maybe it works? The docs are a bit confusing in the examples.

Reply:
Oh, huh, I thought it was globs patterns for the imports. I _think_ we leverage this elsewhere. We'd have to confirm though. If it doesn't work, please disregard the suggestion :)